1-exercises/E-array-map/exercise.js

```diff
@@ -3,7 +3,9 @@
 
 let numbers = [0.1, 0.2, 0.3, 0.4, 0.5];
 
-let numbersMultipliedByOneHundred; // complete this statement
+let numbersMultipliedByOneHundred = numbers.map( function double(number){
+  return number * 100;
+}); // complete this statement
```

Comment:
this is fine, however for ease of readability I'd recommend either creating your function definition outside of the `.map` and referencing it here (e.g. `numbers.map(multiplyByOneHundred)`) or using an anonymous arrow function (e.g. `numbers.map((x) => x*100)`).